src/components/ToolbarButton.tsx

```diff
@@ -165,7 +165,7 @@ const ToolbarButton: FC<ToolbarButtonProps> = ({
         // offset top to avoid changing container height
         // marginBottom: isPressing ? -10 : 0,
         // top: isButtonExecutable && isPressing ? 10 : 0,
-        transform: `translateY(${isButtonExecutable && isPressing && !longPress.isPressed && !isDragging ? 0.25 : 0}em`,
+        transform: `translateY(${isButtonExecutable && isPressing && !longPress.isPressed && !isDragging && !showColorPicker ? 0.25 : 0}em`,
```

Comment:
I would highly recommend leaving this as-is. The toolbar press uses very carefully designed behavior implemented in #2214.